tests/pipeline/test_downsampling.py

```diff
@@ -684,3 +689,80 @@ def test_errors_on_bad_downsample_frequency(self):
             "for argument 'frequency', but got 'bad' instead."
         ).format(_qualified_name(f.downsample))
         self.assertEqual(str(e.exception), expected)
+
+
+class TestDownsampledRowwiseOperation(WithAssetFinder, ZiplineTestCase):
+    T = partial(pd.Timestamp, tz='utc')
+    START_DATE = T('2014-01-01')
+    END_DATE = T('2014-02-01')
+    HALF_WAY_POINT = T('2014-01-15')
+
+    dates = pd.date_range(START_DATE, END_DATE)
+    factor = TestingDataSet.float_col.latest
+
+    @classmethod
```

Comment:
Is there a particular reason that we want this test to use `populate_initial_workspace` to produce `factor`?  Another way we've produced this same factor in tests elsewhere has been to define a factor like:

```python
class Sid(CustomFactor):
    inputs = []
    window_length = 1
    def compute(self, today, assets, out):
        out[:] = assets
```